x-pack/plugins/security_solution/cypress/e2e/exceptions/entry/comments.cy.ts

```diff
@@ -0,0 +1,259 @@
+
+describe('Add, copy comments in different exceptions type and validate sharing them between users', () => {
+  describe('Rule exceptions', () => {
+    before(() => {
+      esArchiverResetKibana();
+      esArchiverLoad('exceptions');
+      login();
+      deleteAlertsAndRules();
+      const exceptionList = getExceptionList();
+      // create rule with exceptions
+      createExceptionList(exceptionList, exceptionList.list_id).then((response) => {
+        createRule({
+          ...getNewRule(),
+          query: 'agent.name:*',
+          index: ['exceptions*'],
+          exceptions_list: [
+            {
+              id: response.body.id,
+              list_id: exceptionList.list_id,
+              type: exceptionList.type,
+              namespace_type: exceptionList.namespace_type,
+            },
+          ],
+          rule_id: '2',
+        });
+        createExceptionListItem(exceptionList.list_id, {
+          list_id: exceptionList.list_id,
+          item_id: 'simple_list_item',
+          tags: [],
+          type: 'simple',
+          description: 'Test exception item 2',
+          name: 'Sample Exception List Item 2',
+          namespace_type: 'single',
+          entries: [
+            {
+              field: 'unique_value.test',
+              operator: 'included',
+              type: 'match_any',
+              value: ['foo'],
+            },
+          ],
+        });
+      });
+    });
+    beforeEach(() => {
+      visitWithoutDateRange(DETECTIONS_RULE_MANAGEMENT_URL);
+      goToRuleDetails();
+      goToExceptionsTab();
+    });
+
+    after(() => {
+      esArchiverUnload('exceptions');
+    });
+
+    it('Add comment on a new exception, add another comment has unicode from a different user and copy to clipboard', () => {
+      cy.get(NO_EXCEPTIONS_EXIST_PROMPT).should('not.exist');
+
+      // User 1
+      // open add exception modal
+      addExceptionFlyoutFromViewerHeader();
+
+      cy.get(LOADING_SPINNER).should('not.exist');
+
+      // add exception item conditions
+      addExceptionConditions(getException());
+
+      // add exception item name
+      addExceptionFlyoutItemName('My item name');
+
+      // add exception comment
+      addExceptionComment('User 1 comment');
+
+      // submit
+      submitNewExceptionItem();
+
+      // new exception item displays
+      cy.get(EXCEPTION_ITEM_VIEWER_CONTAINER).should('have.length', 2);
+
+      // click on show comments
+      clickOnShowComments();
+      // copy the first comment to clipboard
+      clickCopyCommentToClipboard();
+
+      cy.get(EXCEPTION_ITEM_COMMENTS_CONTAINER_TEXT).eq(0).should('have.text', 'User 1 comment');
+
+      // User 2
+      // Login with different users to validate accessing comments of different users
+      login(ROLES.detections_admin);
+
+      // Navigate to Rule page
+      visitWithoutDateRange(DETECTIONS_RULE_MANAGEMENT_URL);
+      goToRuleDetails();
+
+      goToRuleDetails();
```

Comment:
I believe we can delete this method since looks like a duplication from the above in line 143.